### PR TITLE
Response

### PR DESCRIPTION
**Pull request by lisamsmith100**
I got kicked off my response branch and so I recommitted, repushed and repulled

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -7,40 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
...
-<!-- Remove this comment and place your answer here. -->
+<!-- git checkout -b response OR git branch response then git checkout response -->
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git diff -->
```
Comment (jordanallain) [inline diagnostic.md:29]: Very close here Lisa! `git diff` will actually show us all of the changes we have made from a file while `git status` will give us a summary of which files have been changed.

File: diagnostic.md [hunk 2]
```diff
@@ -7,40 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
...
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git add diagnostic.md -->
 ```
 
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
-<!-- Remove this comment and place your answer here. -->
+<!-- first letter in message = capitalized; first word = verb (imperative)   -->
```
Comment (jordanallain) [inline diagnostic.md:42]: Really close here too Lisa. The two formatting items we need are a short title that summarizes what we've changed, and then a longer message beneath the title that describes in more detail what we've changed.

As far as your answer though you were right in that is the way we want our commit title's to be structured. Good job.

**Comment by jordanallain** [comment]
7/9

Really great work Lisa! You got snagged up on a couple but you were really close in those and the rest of them you were right on. Let us know if you need extra clarification on anything but keep up the great effort.

🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴

**Comment by lisamsmith100** [comment]
Ah yes.  I write git status after everything.  Thanks!

**Pull request closed without merging**